internal/storage/driver/memory/binding.go

```diff
@@ -61,3 +61,15 @@ func (s *Binding) ListByInstanceID(instanceID string) ([]internal.Binding, error
 
 	return bindings, nil
 }
+
+func (s *Binding) Get(instanceID string, bindingID string) (*internal.Binding, error) {
```

Comment:
Current in-memory implementation does not support multiple bindings per instance.

Reply:
We do not support such case. We only need to return one binding at a time.

Reply:
We do.

Reply:
If we limit number of bindings per instance to 10 we allow more than 1 binding per instance.

Reply:
This comment was invalid. We support it in both drivers.